theme/less/components/toast.less

```diff
@@ -0,0 +1,133 @@
+.fadeOut {
```

Comment:
I think the toast styling fails. It shows like this.
![image](https://user-images.githubusercontent.com/28541902/171028628-a6def57f-41c3-4b2a-9c7e-7b79e62207a8.png)

Reply:
Fixed !